src/main_dpdk.cpp

```diff
@@ -1959,8 +1961,10 @@ HOT_FUNC int CCoreEthIFStateless::send_node_flow_stat(rte_mbuf *m, CGenNodeState
     lp_s->add_bytes(mi->pkt_len + 4); // We add 4 because of ethernet CRC
 
     if (hw_id >= MAX_FLOW_STATS) {
-        fsp_head->time_stamp = CGlobalInfo::m_options.m_get_latency_timestamp();
-
+        fsp_head->time_stamp = CGlobalInfo::m_options.get_latency_timestamp();
+        if (CGlobalInfo::m_options.is_timesync_enabled()) {
+            fsp_head->time_stamp = (uint64_t)(CGlobalInfo::get_timesync_engine()->getDelta(lp_port->m_port->get_tvpid()) + (int64_t)fsp_head->time_stamp);
```

Comment:
We can use here += and change C cast to static_cast

Reply:
done in bd1aed19c572e9eb48e42a37603797844290c27c